doc-requirements.txt

```diff
@@ -1,6 +1,8 @@
 # Requirements for building docs
 -r requirements.txt
 sphinx<3
-numpydoc
+jinja2<3
```

Comment:
I needed to pin these to get version compatibility to work. In the future might be better to use a tool like poetry so that we can generate a lockfile and not have to deal with this sort of not-fun dependency pinning

Reply:
I needed this to get my local to pass but it seems like CI might not like it.... I'll try reverting to see if if this is what is breaking the pre-release CI jobs